main.py

```diff
@@ -3,10 +3,8 @@
 
 
 def main():
-    methods.startGame()
-    num = methods.setNumber()
-    lives = methods.setLives()
-    methods.compareNumber(num, lives)
+    methods.startGame()  # do you want to start yes or no?
```

Comment:
might want to rename methods module to something else. methods is really generic, and you want to be as specific as possible. It's basically a game engine, right? maybe name it as such. engine.startGame() Since it's a module you might want to build it out as a class as well. Object Oriented is the way to go when you can help it.